config/gazebo9_bionic.yaml

```diff
@@ -0,0 +1,6 @@
+name: gazebo9bionic
+method: https://launchpad.net/~j-rivero/+archive/ubuntu/gazebo9bionic
```

Comment:
```suggestion
method: http://ppa.launchpad.net/j-rivero/gazebo9bionic/ubuntu
```

The method needs to be the apt repository URL not the web link to the PPA.

Reply:
ouch, thanks.